docs/html/user_guide.rst

```diff
@@ -1150,3 +1150,80 @@ announcements on the `low-traffic packaging announcements list`_ and
 .. _our survey on upgrades that create conflicts: https://docs.google.com/forms/d/e/1FAIpQLSeBkbhuIlSofXqCyhi3kGkLmtrpPOEBwr6iJA6SzHdxWKfqdA/viewform
 .. _the official Python blog: https://blog.python.org/
 .. _Python Windows launcher: https://docs.python.org/3/using/windows.html#launcher
+
+Using system trust stores for verifying HTTPS in 22.2 (2022)
+============================================================
+
+pip 22.2 added **experimental** support for using system trust stores to verify HTTPS certificates
+instead of certifi. Using system trust stores has advantages over certifi like automatically supporting
+corporate proxy certificates without additional configuration.
+
+In order to use system trust stores you must be using Python 3.10+ and install the package `truststore`_ from PyPI.
+
+.. tab:: Unix/macOS
+
+   .. code-block:: console
+
+      # Requires Python 3.10 or later
+      $ python --version
+      Python 3.10.4
+
+      # Install the 'truststore' package from PyPI
+      $ python -m pip install truststore
+      [...]
+
+      # Use '--use-feature=truststore' flag to enable
+      $ python -m pip install SomePackage --use-feature=truststore
+      [...]
+      Successfully installed SomePackage
+
+.. tab:: Windows
+
+   .. code-block:: console
+
+      # Requires Python 3.10 or later
```

Comment:
Windows command prompt uses `%` for comment. But this is not actually intended to be directly pasted anywhere, so I guess it’s fine.

Reply:
Let me know if you want me to change this or if there's anything else blocking a merge. Thanks!